theapp/static/js/chat.js

```diff
@@ -34,38 +35,32 @@ var chosen_color = getRandomColor();
 
 
 var form = $('form').on('submit', function ( event ){
-	// to allow for when the user hits enter
-	event.preventDefault();
-	var message   = $('input.message').val();
-
-	socket.emit( 'chat', {
-		user : alias,
-		msg : message,
-		color: chosen_color
-	});
-
-	console.log(alias, message, chosen_color);
-
-	//Empty field
-
+    // to allow for when the user hits enter
+    event.preventDefault();
+    var message   = $('input.message').val();
+    var path = window.location.pathname.split('/');
+    room = path[2];
+
+    socket.emit( 'message', {
+        user : alias,
+        msg : message,
+        color: chosen_color,
+        room: room
+    });
+    $('#chatinputbox').val('');
 });
 
 $(window).on('resize',function() {
-	$('#chatbox').css('max-height',$(window).height() - 150);
+    $('#chatbox').css('max-height',$(window).height() - 150);
 });
 
-
-
-
-// Capture Message
-socket.on('chat', function( msg ){
-	// FIX THIS msg.user
-	if( typeof msg.user !== 'undefined' ){
-		$('h1').remove();
-		$('div.message_holder').append('<div class="message_roll"><b style="color:' + msg.color + '">' + msg.user + ': </b>' + msg.msg + '</div>' );
-		$('#chatbox').scrollTop($('#chatbox')[0].scrollHeight);
-		$('#chatbox').css('max-height', $(window).height() - 150);
-		// clear textbox
-		$('#chatinputbox').val('');
```

Comment:
No, the functionality has been moved.  This is the code that caused bugs in our demo.